scripts/cleanup_edomain_templates.py

```diff
@@ -49,6 +50,24 @@ def make_ssh_client(rhevip, sshname, sshpass):
     return SSHClient(**connect_kwargs)
 
 
+def is_pingable(ip_address):
```

Comment:
Couple of thoughts:

* Could this live in `utils.net`? I think we already have some function to check if a port is open, maybe that could be used instead if you know the port on which RHEV communicates.
* The comment has the wrong doc format :) Check out eg. `cfme.web_ui` for the example.